docker-compose.yml

```diff
@@ -7,9 +7,7 @@ services:
       - MYSQL_USER=remo
       - MYSQL_PASSWORD=remo
       - MYSQL_ROOT_PASSWORD=root
-    entrypoint:
-      - docker-entrypoint.sh
-      - --max_allowed_packet=2M
+    command: mysqld --max_allowed_packet=64M
```

Comment:
I see the file exist in the repository
https://github.com/docker-library/mysql/tree/e8a0ed55678d27039905c50f8327f3a367e92498/5.5